app/views/layouts/application.html.erb

```diff
@@ -6,13 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= javascript_pack_tag "application" %>
     <%= stylesheet_pack_tag "application" %>
   </head>
 
   <body class="<%= "#{controller.controller_name} #{@body_class}".strip %> h-full">
     <main class="w-full h-full">
       <%= yield %>
     </main>
+
+    <%= javascript_pack_tag "application" %>
```

Comment:
Load async in top

Reply:
Load "defer" in top was the answer, and also what webpacker does by default :champagne: